InstabugSample/src/screens/SettingsScreen.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { Switch, StyleSheet, Text, View } from 'react-native';
+import Instabug, { BugReporting } from 'instabug-reactnative';
+
+import Button from '../components/Button';
+import ColorButton from '../components/ColorButton';
+import Screen from '../components/Screen';
+import Section from '../components/Section';
+
+const colors = ['crimson', 'olivedrab', 'cornflowerblue', 'gold'];
+
+function SettingsScreen() {
+  const [isLightMode, setIsLightMode] = useState(true);
+  const [currentColor, setCurrentColor] = useState('');
+
+  const toggleColorTheme = _isLightMode => {
+    setIsLightMode(_isLightMode);
+    Instabug.setColorTheme(_isLightMode ? Instabug.colorTheme.light : Instabug.colorTheme.dark);
+  };
+
+  const setPrimaryColor = color => {
+    Instabug.setPrimaryColor(color);
+    setCurrentColor(color);
+  };
+
+  const changeInvocationEvent = invocationEvent =>
+    BugReporting.setInvocationEvents([invocationEvent]);
+
+  return (
+    <Screen>
+      <Section title="Invocation Event">
+        <View style={[styles.row, { marginTop: -10 }]}>
+          <Button onPress={() => changeInvocationEvent(BugReporting.invocationEvent.shake)}>
+            Shake
+          </Button>
+          <Button onPress={() => changeInvocationEvent(BugReporting.invocationEvent.screenshot)}>
+            Screenshot
+          </Button>
+          <Button
+            onPress={() => changeInvocationEvent(BugReporting.invocationEvent.twoFingersSwipe)}>
+            Two Fingers Swipe Left
+          </Button>
+          <Button
+            onPress={() => changeInvocationEvent(BugReporting.invocationEvent.floatingButton)}>
+            Floating Button
+          </Button>
+          <Button onPress={() => changeInvocationEvent(BugReporting.invocationEvent.none)}>
+            None
+          </Button>
```

Comment:
Sounds like a good idea to me. But let's make the string value the key instead of the enum value (even if enum values might be a string internally but just to avoid inconvenience) as object literal keys in JavaScript can only be strings (they are implicitly converted to strings if they aren't already).

```js
{
  Shake: BugReporting.invocationEvent.shake
}
```

Reply:
Another solution would be to have an array of objects like this:

```js
const invocationEvents = [
  { label: 'Two fingers swipe left', value: BugReporting.invocationEvent.twoFingersSwipe },
  ...
];
```

this will allow us to avoid spaces in object keys if we use the Map-like approach which will look like this:

```js
const invocationEvents = {
  'Two fingers swipe left': BugReporting.invocationEvent.twoFingersSwipe,
  ...
}
```

There's nothing wrong with this approach other than preference and code style in the end, what do you think?

Reply:
I like the one you suggested as well, let's go with it!